src/app/global/app_sections_config.nim

```diff
@@ -24,3 +24,7 @@ const SETTINGS_SECTION_ICON* = "settings"
 const SWAP_SECTION_ID* = "swap"
 const SWAP_SECTION_NAME* = "Swap"
 const SWAP_SECTION_ICON* = "swap"
+
+const TRADING_CENTER_SECTION_ID* = "trading"
+const TRADING_CENTER_SECTION_NAME* = "Trading"
```

Comment:
This is not translated by default; pls add the respective cases to `Utils.translatedSectionName()`

Reply:
Swap too pls